lib/tasks/hbx_reports/enrollment_report.rake

```diff
@@ -117,7 +110,7 @@ namespace :reports do
                   per.ethnicity,
                   ethnicity_status(per.ethnicity),
                   per.citizen_status,
-                  broker_assisted(enr, primary_person)
+                  family.current_broker_agency&.writing_agent.present? ? "Yes" : "No"
```

Comment:
Are these english strings that appear to the end user?

Reply:
Nope - this is just a report for CMS